### PR TITLE
DO/CS spelling

### PR DESCRIPTION
**Pull request by snuyanzin**
The PR suggests to fix a bunch of misprints which I faced going through docs
atleast       =>  at least 
precednce => precedence
optained => obtained 
defualt => default
configuation => configuration 
[Taefik](https://docs.traefik.io) => [Traefik](https://docs.traefik.io)
Servicess  => Services
artficats => artifacts 
recommnedations => recommendations
confiagurations => configurations
certificte => certificate 
neworks => networks 
explicity => explicitly 
occured => occurred
opeprates => operates
explaination => explanation 
evironment => environment 
sitations => citations 
trustore => truststore 
contianing => containing 
colocated  => collocated 
enteries => entries
pluging => plugging 
hese  => these
commants  => commands 
updted => updated
faciliate  => facilitate 
```
<config>
  <port>1984</port>
  <game>mysvc</game>
  <!-- ... -->
<config>
``` 
=>
 ```
<config>
  <port>1984</port>
  <game>mysvc</game>
  <!-- ... -->
</config>
```

### CODE REVIEW

File: docs/pages/yaml-reference.md [hunk 1]
```diff
@@ -380,7 +380,7 @@ This documentation effectively reflects the Java object tree under [RawServiceSp
 
     * `health-check`
 
-      Health checks are additional validation that your task is healthy, in addition to just the fact that its process is still running. This is an extra convenience for sitations where a service can enter a zombie state from which it can never return. For example, it might query an HTTP endpoint to validate that an HTTP service is still responding.
+      Health checks are additional validation that your task is healthy, in addition to just the fact that its process is still running. This is an extra convenience for citations where a service can enter a zombie state from which it can never return. For example, it might query an HTTP endpoint to validate that an HTTP service is still responding.
```
Comment (kaiwalyajoshi) [inline docs/pages/yaml-reference.md]: I think this was intended to be corrected to `situations` rather than `citations`

Comment (snuyanzin) [reply docs/pages/yaml-reference.md]: Agree. Thank you

**Comment by snuyanzin** [comment]
Fixed correction for `sitations` pointed by @kaiwalyajoshi in a separate commit.
Please let me know if squash is required

**Comment by kvish** [comment]
@kaiwalyajoshi this looks good to me any objection to merging it?, since this is just documentation related no need to wait on CI to pass. 

@snuyanzin thanks so much for the help!

**Pull request merged**